components/workflow.rst

```diff
@@ -79,10 +79,11 @@ Usage
 -----
 
 When you have configured a ``Registry`` with your workflows,
-you can retrieve a workflow from it and use it as follows::
+you can retreive a workflow from it and use it as follows::
 
     // ...
     // Consider that $blogPost is in place "draft" by default
+    // Consider that $post is in state "draft" by default
```

Comment:
The variable `$post` doesn't exist.

Reply:
yes i updated it in another PR, and on the original one i added this line
which then create a fail merge :/